test/ex_webrtc/rtp/jitter_buffer_test.exs

```diff
@@ -0,0 +1,127 @@
+defmodule ExWebRTC.RTP.JitterBufferTest do
```

Comment:
I feel like we inspect packet store in those tests too many times. 

E.g. instead of using PacketStore.dump use JitterBuffer.flush, instead of PacketStroe.insert_packet use JitterBuffer.place_packet

Reply:
Changed a few of the calls to `PacketStore`, but ultimately I'm unsure if this is the way to go

Reply:
The idea is to test JitterBuffer API. In other case, any change in PacketStore API will cause these tests to fail. Sometimes, indeed, we want to check some internal field but we should avoid that as much as possible. In the case of JitterBuffer, I think that we can always find a sequence of packets that after insertion into JitterBuffer  will set PacketStore state in a desired way.